File: 171026_C/031.c
// 031 
// Resting Pulse: 65 Age:22
// Intensity	| Rate
// -------------|-----------
// 55%			| 138bpm

#include <stdio.h>
#include <stdlib.h>


int main() {
	int pulse, age;
	double targetHeartRate, intensity;

	printf("Resting pulse: ");
	scanf("%d", &pulse);
	printf("Age: ");
	scanf("%d", &age);
	printf("Intensity\t| Rate\n");
	printf("----------------|--------\n");
	


	for (intensity = 55; intensity <= 95; intensity += 5) {
		targetHeartRate = (((220 - age) - pulse)*intensity/100) + pulse;
		printf("%.0lf%%\t\t| %.0lfbpm\n", intensity, targetHeartRate);
	}

	system("pause");
	return 0;


}

File: 171108_Python/031.py

# 031
# Resting Pulse: 65 Age:22
# Intensity	| Rate
# -------------|-----------
# 55%			| 138bpm

restingPulse = int(input("Resting Pulse: "))
age = int(input("Age: "))

print("Intensity\t | Rate")
print("-" * 12, "|", "-" * 10)

for i in range(11, 20):
	targetHR = (((220 - age) - restingPulse) * i / 20) + restingPulse
	print("%d%%\t\t\t | %d bpm" % (i * 5, targetHR))
